packages/suite/src/config/suite/features.ts

```diff
@@ -6,6 +6,7 @@
 // General flags
 export const FLAGS = {
     GOOGLE_DRIVE_SYNC: false, // Google Drive sync (used for labeling)
+    RBF: false,
```

Comment:
There should be a description of what the flag is about.

Reply:
updated